ui/DesignSystem/Component/Notification.svelte

```diff
@@ -6,6 +6,10 @@
   export let notification: Notification;
   export let style = "";
 
+  const variantClass = notification.styleVariant
+    ? `variant-${notification.styleVariant}`
+    : "";
+
```

Comment:
Could we instead just introduce another `level`? This would simplify the setup (only one parameter influencing the color)

Reply:
You’re right. I think this is a better approach. But then I’d move away from the term “level” and use “variant” because it’s usually used for logging levels.

@brandonhaslegs do you have a better suggestion than calling the variant “primary”?